### PR TITLE
NOISSUE - update default service ports

### LINKED ISSUE #140: NOISSUE - WS port wrong on code example
**BUG REPORT**

1. What were you trying to achieve?
Send a message over websockets using golang example

3. What are the expected results?
successful message sent

5. What are the received results?
Error connecting to Websocket Server: dial tcp [::1]:8190: connect: connection refused

7. What are the steps to reproduce the issue?
https://github.com/mainflux/docs/blob/1b598be9ed63f8bc56a881219a477b5452950c18/docs/messaging.md?plain=1#L143

Port should be 8186
8. In what environment did you encounter the issue?

9. Additional information you deem important:

### PR DESCRIPTION
**Pull request by SammyOina**
### What does this do?
Update service ports in docs for mainflux v0.14
### Which issue(s) does this PR fix/relate to?
Fixes #140 
Refers to [PR](https://github.com/mainflux/mainflux/pull/1737)

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
No

### CODE REVIEW

**Comment by dborovcanin** [comment]
Is this one ready? After this one, ofc: https://github.com/mainflux/mainflux/pull/1737

**Comment by SammyOina** [comment]
> Is this one ready? After this one, ofc: [mainflux/mainflux#1737](https://github.com/mainflux/mainflux/pull/1737)

@dborovcanin Yes, this ready for review, but I'll move it from draft once [mainflux/mainflux#1737](https://github.com/mainflux/mainflux/pull/1737) is merged

**Review by dborovcanin — DISMISSED** [review]

**Comment by dborovcanin** [comment]
We need updates on [Devops repo](https://github.com/mainflux/devops/pulls) before merging this one.

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
update:
- [ ] docs/bootstrap.md
- [ ] docs/edge.md
- [ ] docs/storage.md
- [ ] docs/twins.md

**Comment by SammyOina** [comment]
> We need updates on [Devops repo](https://github.com/mainflux/devops/pulls) before merging this one.

PR open [here](https://github.com/mainflux/devops/pull/123)

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Update `docs/messaging.md` and `docs/provision.md`

**Review by dborovcanin — DISMISSED** [review]

**Review by rodneyosodo — CHANGES_REQUESTED** [review]
Change `localhost:8204` in `docs/provision.md`
also in  `docs/storage.md`

File: docs/storage.md [hunk 1]
```diff
@@ -193,7 +194,7 @@ curl -s -S -i  -H "Authorization: Thing <thing_key>" http://localhost:<service_p
 If you don't provide `offset` and `limit` parameters, default values will be used instead: 0 for `offset` and 10 for `limit`.
 The `format` parameter indicates the last subtopic of the message. As indicated under the [`Writers`](storage.md#writers) section, the message format is stored in the subtopic as the last part of the subtopic. In the example:
 ```
-http://localhost:8185/channels/<channelID>/messages/home/temperature/myFormat
+http://localhost:8008/channels/<channelID>/messages/home/temperature/myFormat
```
Comment (rodneyosodo) [inline docs/storage.md]:
```suggestion
http://localhost:<service_port>/channels/<channelID>/messages/home/temperature/myFormat
```

File: docs/storage.md [hunk 2]
```diff
@@ -127,7 +128,7 @@ _Please note that Cassandra may not be suitable for your testing environment bec
 
 ### MongoDB and MongoDB Writer
 
-```bash
+```bas8202h
```
Comment (rodneyosodo) [inline docs/storage.md]:
```suggestion
```bash
```

File: docs/storage.md [hunk 3]
```diff
@@ -113,7 +113,8 @@ This will install and start:
 Those new services will take some additional ports:
 
 - 8086 by InfluxDB
-- 8900 by InfluxDB writer service
+- 9006 by InfluxDB writer service
+- 3001 by Grafana
```
Comment (rodneyosodo) [inline docs/storage.md]: This was removed by https://github.com/mainflux/docs/pull/139

Comment (rodneyosodo) [inline docs/storage.md]:
```suggestion
```

**Review by rodneyosodo — APPROVED** [review]

**Review by drasko — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**